internal/summary/summary.go

```diff
@@ -24,29 +33,87 @@ func NewSummary() *Summary {
 }
 
 // Gather collates data into Summary
-func (s *Summary) Gather(services []model.Service) {
+func (s *Summary) Gather(services []model.Service) error {
 
 	for _, svc := range services {
-		var path string
 		// Assume that, in practice, there is only one mount
+		path := ""
 		if len(svc.Mounts) > 0 {
 			path = svc.Mounts[0].Repo.LocalPath
-		} else {
-			path = ""
 		}
-		s.Services = append(s.Services, &Service{
-			Name: string(svc.Name),
-			Path: path,
-		})
+
+		svcSummary := &Service{
+			Name:    string(svc.Name),
+			Path:    path,
+			k8sData: []k8sDatum{},
+		}
+
+		kubeDatum := &k8sDatum{}
+
+		entities, err := k8s.ParseYAMLFromString(svc.K8sYaml)
+		if err != nil {
+			return err
+		}
+
+		lbs := k8s.ToLoadBalancers(entities)
+
+		for _, lb := range lbs {
+			kubeDatum.LoadBalancer = append(kubeDatum.LoadBalancer, lb)
+		}
+
+		for _, e := range entities {
+			kubeDatum.Group = e.Kind.Group
+			kubeDatum.Version = e.Kind.Version
```

Comment:
...will the entries ever have a different Group or Version from each other? in this code, your datum will only hang onto the group/version of the very last entry you come upon

Reply:
Oh, this is a very good point! I see the logic error here now. I think I will hastily merge in this commit just to keep barreling forth, but I made a note of this issue to investigate further. It may also be the case that I don't really need this info for my summary, anyhow.